webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -2166,6 +2167,25 @@ def is_percentage(thing):
         ok_(is_percentage(first_row[3]))  # throttle
         ok_(is_percentage(first_row[4]))  # ratio
 
+    def test_crashes_per_user_legacy_by_build_date(self):
+        url = reverse('crashstats:crashes_per_user')
+
+        response = self.client.get(url, {
+            'date_range_type': 'build',
+            'product': 'Whatever',
+            'foo': 'bar'
+        })
+        # If we use self.assertRedirects() it will actually go to the
+        # redirected URL which we haven't set up mocking for yet.
+        eq_(response.status_code, 302)
+        redirect_url = response['Location']
+        parsed = urlparse.urlparse(redirect_url)
+        eq_(parsed.path, url)
+        ok_('product=Whatever'in parsed.query)
+        ok_('foo=bar'in parsed.query)
+        ok_('date_range_type=build' not in parsed.query)
+        ok_('date_range_type=' not in parsed.query)
```

Comment:
Just for the sake of arguing: if ``date_range_type=`` is not in ``parsed.query``, ``date_range_type=build`` definitely is not in it either. :) So the second line covers both cases, and the first one is unneeded.